### DISCUSSION TITLE
Distinguishing how data was gathered

### DISCUSSION THREAD

**Discussion by jac0-b** [Anarios/return-youtube-dislike#391] [Ideas]
It seems is going to three different sources of data and it makes sense to have some sort of way to tell users how the data for the video they are watching. 

My suggestion would be a coloured badge or like/dislike numbers e.g.
- Green for youtube studio data (most accurate) 
![image](https://user-images.githubusercontent.com/51512690/147796845-644bca8d-c1ad-4533-b8a9-d1ffa4369b27.png)
- Orange for historical data
![image](https://user-images.githubusercontent.com/51512690/147796872-5adedc0a-70a7-49b2-8fcd-d7f49d376b5f.png)
- Red for user data (least accurate)
![image](https://user-images.githubusercontent.com/51512690/147796904-bcafd610-f0f0-4b0e-a8dd-532565dcb98b.png)

**Comment by Hans5958** [comment]
~~How about show it in a different way such as a thinner line at the bottom of the ratio line?~~

#286 has the same discussion

**Comment by sy-b** [comment]
Related to
Open - #108 #114 #286 &
Closed - #353 #221
